client/packages/system/src/Patient/ProgramEnrolment/api/operations.graphql

```diff
@@ -16,6 +16,14 @@ fragment Document on DocumentNode {
   }
 }
 
+fragment ProgramRow on ProgramNode {
+  enrolmentDatetime
```

Comment:
also not required. wanted to reduce the amount of data returned for the list view down to the minimum. The documentRegistry is needed for the list